stream-chat-android-ui-components/src/main/res/drawable/stream_ui_ic_down.xml

```diff
@@ -5,7 +5,7 @@
     android:viewportHeight="24"
     >
     <path
-        android:fillColor="#000000"
+        android:fillColor="@color/stream_ui_black"
```

Comment:
Was it intentional? `stream_ui_black` is white in dark mode

Reply:
The background is black, then it should be white so this icon appears